### PR TITLE
feat: arkconnect navbar button

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

https://app.clickup.com/t/86drwqqf8

requires https://github.com/ArkEcosystem/laravel-foundation/pull/613

Todo:

- [x] Mobile state - awaiting designs
- [x] .env option to disable & add a test for it

To test:

- add `ARKCONNECT_ENABLED=true` to `.env`
- make sure you're connecting to a `https` instance of explorer. For valet, you can use `valet secure` to force https - use `valet unsecure` to change it back to `http`

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Review by alfonsobries — CHANGES_REQUESTED** [review]
This doesnt seem right
image

image

**Review by alfonsobries — CHANGES_REQUESTED** [review]
Other than the mode-color the design is not applied properly, design:

image

**Review by alfonsobries — COMMENTED** [review]
Finally the `hasExtension()` condition doesnt seem to work for me (i removed that condition to test this PR)

Tried to install the latest version of the arkconnect extension on chrome+mac and `window.arkconnect` is undefined, same happens if I run the develop version of the plugin, not sure if I missing something

File: webpack.mix.js [hunk 1]
```diff
@@ -25,10 +25,6 @@ mix.webpackConfig({
     // App
     .js("resources/js/app.js", "public/js")
     .js("resources/js/chart-tooltip.js", "public/js")
-    .js(
-        "vendor/arkecosystem/foundation/resources/assets/js/clipboard.js",
-        "public/js/clipboard.js"
-    )
```
Comment (ItsANameToo) [inline webpack.mix.js:31]: ? now it's no longer minimized 🤔

**Review by ItsANameToo — CHANGES_REQUESTED** [review]
also extension doesn't get found locally, even though `window.arkconnect` is available in devtools

**Review by alfonsobries — CHANGES_REQUESTED** [review]
design is good now, but now the button is always disabled

image


Probably because of this line is never setting isLoading to false

https://github.com/ArdentHQ/arkscan/pull/799/files#diff-107b51fb3ed89d5d781341dc4e51b926a020477c468cf86edc0b9d24603e17d7R25-R27

Also in my case I have arkconnect, maybe when that function runs the window.arkconnect  is not set yet

**Comment by alfonsobries** [comment]
working now @alexbarnsley

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**